discord_thread_summarizer_using_openai/ai_utils.bal

```diff
@@ -0,0 +1,19 @@
+import ballerinax/openai.chat;
+
+configurable string openAIToken = ?;
+
+final chat:Client chatClient = check new ({
+    auth: {
+        token: openAIToken
+    }
+});
+
+function generateChatCompletion(string prompt, string model = "gpt-3.5-turbo") returns string|error {
+    chat:CreateChatCompletionRequest req = {
```

Comment:
Do we need a separate variable here?

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/9018e16abefb68f7d46cfbc2a86a00240ce6ea5f